features/facebookImport/src/popUps/infoPopUps/activitiesInfo.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import i18n from "../../i18n.js";
+import BaseInfoPopUp from "../baseInfoPopUp/baseInfoPopUp.jsx";
+import Infographic from "../../components/infographic/infographic.jsx";
+
+import "./infoPopUps.css";
+
+const ActivitiesInfoScreen = () => {
+    const activitiesInfoText = [
+        <>
+            <p>{i18n.t("activitiesInfoScreen:text1")}</p>
+            <div className="legend chart-description-title">
+                {i18n.t("activitiesInfoScreen:legend")}
+            </div>
+            <Infographic
+                type="activitiesBarChart"
+                texts={{
+                    text1: i18n.t("infographics:activitiesBarChart.text1"),
+                    bigbold1: i18n.t(
+                        "infographics:activitiesBarChart.bigbold1"
+                    ),
+                    bold1: i18n.t("infographics:activitiesBarChart.bold1"),
+                    bold2: i18n.t("infographics:activitiesBarChart.bold2"),
+                    boldJan: i18n.t("infographics:activitiesBarChart.boldJan"),
```

Comment:
There's too much boilerplate here, but I guess it's carried over from before, so no big deal.

Reply:
Exactly, this is carried over from before. The intention of this PR is not refactoring this.